class-structure-2024

```diff
@@ -0,0 +1,26 @@
+---
+date: 2024-05-01T09:00:00-05:00
+description: "New class changes starting in May 2024"
+featured_image: '/images/lewisburg.jpg'
+tags: ["weekly dances"]
+title: "New class changes starting in May 2024"
+---
+
+**We are now offering 2 hour long classes starting at 7pm!**
+
+We are excited to share some upcoming changes to our weekly dance schedule! If you've danced with us recently you may have noticed we're growing, which is awesome! To accommodate everyone and better serve a range of skill levels we are restructuring our two weekly classes.
+
+Starting in May and June 2024, we will have an hour-long Fundamentals class for all attendees. This new class content will rotate every two months covering various themes under both 6- and 8-count footwork.
+
+This fundamentals class is meant for individuals who are
+
+* New to Lindy Hop
+* Looking to continue learning after taking a few classes
+* Reviewing fundamentals after a long time away 
+* Interested in solidifying their foundational Lindy Hop skills
+
+If if you have Lindy Hop basics down, Fundamentals might not be for you... and that's why we are offering a second brand new class: “Lindy Hop: Next Steps”!
+
+Next Steps will offer intermediate to advanced students an opportunity to take their core Lindy Hop skills to the next level of technique and complexity as well as expand their repertoires into Charleston and solo jazz topics like musicality, footwork variations, and classic routines.This class will also offer introductions to other swing dance styles such as St Louis Shag, Collegiate Shag, Balboa, or Blues! Because Next Steps will expect students to have solid basics, each month-long series will be a progressive deep dive into a specific topic with the same instructors.
```

Comment:
```suggestion
Next Steps will offer intermediate to advanced students an opportunity to take their core Lindy Hop skills to the next level of technique and complexity as well as expand their repertoires into Charleston and solo jazz topics like musicality, footwork variations, and classic routines. This class will also offer introductions to other swing dance styles such as St Louis Shag, Collegiate Shag, Balboa, or Blues! Because Next Steps will expect students to have solid basics, each month-long series will be a progressive deep dive into a specific topic with the same instructors.
```